### DISCUSSION TITLE
Not working properly, unable to open Virtual machine

### DISCUSSION THREAD

**Discussion by vaibhav200225** [Genymobile/feedback#19] [Genymotion Desktop] [answered]
Hello team, I am fed-up with this problem after looking many solution on internet it doesn't fix my problem. Why in windows 11 the genymotion not able to start the virtual device. It shows unable to start the virtual machine. Please come with any solution ASAP.

**Comment by agavignet** [comment, marked as answer]
Hi @vaibhav200225 

Could you contact our support team here https://support.genymotion.com? We will need you to provide logs for further investigation.
Also have you look at our FAQ https://support.genymotion.com/hc/en-us/articles/4415611962897-Windows-Unable-to-start-the-Virtual-Device ?

Thank you,
Best regards
